spec/tracker/tracker_spec.rb

```diff
@@ -12,6 +12,11 @@ class BodyHandler < DummyHandler
   self.position = :body
 end
 
+class BodyOpeningHandler < DummyHandler
+  self.position = :body
+  self.exact_position = :opening
```

Comment:
done now with container_tag and container_position. I avoided changing the position attribute at first to avoid changing it everywhere.